### PR TITLE
Update routing.md

### PR DESCRIPTION
**Pull request by njleonzhang**

### CODE REVIEW

File: zh/routing.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function createApp () {
     router,
     render: h => h(App)
   })
-  // 注入 router 到根 Vue 实例
```
Comment (kazupon) [inline zh/routing.md]: why do you remove the comment?

See the `en` docs:
https://github.com/vuejs/vue-ssr-docs/blame/master/en/routing.md#L44

Comment (njleonzhang) [reply zh/routing.md]: I thought it is a redundancy. Because it is same to line 35. But according to the en version, it should be a translation error.

Comment (Jinjiang) [reply zh/routing.md]: Yes, the 2 comments have different meanings imo. The L39 is about inject and L39 is about return. So agree with @kazupon .

File: zh/routing.md [hunk 2]
```diff
@@ -36,8 +36,8 @@ export function createApp () {
...
-  return { app, router }
+  // 返回app和router
```
Comment (Jinjiang) [inline zh/routing.md]: Btw. Please leave spaces between Chinese chars and English letters. (like: `返回 app 和 router`)

**Comment by kazupon** [comment]
Thanks!

**Pull request merged**